packages/frontend-2/lib/projects/composables/tokenManagement.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  convertThrowIntoFetchResult,
+  getFirstErrorMessage,
+  modifyObjectField,
+  getCacheId
+} from '~~/lib/common/helpers/graphql'
+import {
+  deleteEmbedTokenMutation,
+  createEmbedTokenMutation,
+  deleteAllProjectEmbedTokensMutation
+} from '~~/lib/projects/graphql/mutations'
+import { ToastNotificationType, useGlobalToast } from '~~/lib/common/composables/toast'
+import { useApolloClient } from '@vue/apollo-composable'
+
+export const useDeleteEmbedToken = () => {
+  const { triggerNotification } = useGlobalToast()
+  const apollo = useApolloClient().client
+
+  return async (input: { projectId: string; token: string }) => {
+    const { projectId, token } = input
+
+    const result = await apollo
+      .mutate({
```

Comment:
u can leave it as  is, but generally we should prefer using `useMutation` cause it enables loading-tracking composables like useIsMutationLoading